nncf/torch/binarization/binarize_functions.py

```diff
@@ -19,8 +19,24 @@
 from nncf.torch.utils import add_domain
 
 from .extensions import BinarizedFunctionsCUDA
-from torch.onnx.symbolic_helper import _unsqueeze_helper  # pylint:disable=protected-access
-
+from torch.onnx.symbolic_helper import _is_constant  # pylint:disable=protected-access
+
+
+def _unsqueeze_helper(g, input_, axes_i):
+    # Unsqueeze handling for different opsets inspired by torch.onnx.symbolic_helper._unsqueeze_helper
+    # The original unsqueeze_helper cannot be used in 1.13 since it references
+    # an `.opset` attribute on the `g` argument which is not there. The original intent
+    # of this in pytorch was to allow accessing opset info in the symbolic fn's
+    # code (see PR #84728 in the pytorch repo), but somehow the `symbolic` functions
+    # that we define are called from the C++ code, and they pass the old torch._C.Graph
+    # object as `g` instead of the torch.onnx._internal.jit_utils.GraphContext as it
+    # should be.
+    # The effect is that we can only do opset>=13-style unsqueeze here and hope that
+    # the user did not request an older opset.
```

Comment:
Some tests are still using opset lower than 13. Does it make sense to keep logic for lower opset versions?

Reply:
We really only need this logic for our own unsqueezes in binarization. I don't think that there are that many uses for binarization anyway either in test or among users.